src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -272,14 +307,13 @@
                         (swap! state assoc :entity-types entity-types))
                       ;; FIXME: :data-attribute-load-error is unused
                       (swap! state assoc :data-attribute-load-error status-text)))}))
-     (let [{:keys [loaded-config inputs-outputs]} @state]
-       (swap! state assoc :editing? true :original-config loaded-config :original-inputs-outputs inputs-outputs)))
+     (let [{:keys [loaded-config inputs-outputs redacted?]} @state]
+       (swap! state assoc :editing? true :original-config loaded-config :original-inputs-outputs inputs-outputs :original-redacted? redacted?)))
    :-cancel-editing
    (fn [{:keys [state refs]}]
-     (let [original-loaded-config (:original-config @state)
-           original-inputs-outputs (:original-inputs-outputs @state)
-           method-ref (-> original-loaded-config :methodConfiguration :methodRepoMethod)]
-       (swap! state assoc :editing? false :loaded-config original-loaded-config :inputs-outputs original-inputs-outputs)
+     (let [{:keys [original-inputs-outputs original-redacted? original-config]} @state
+           method-ref (-> original-config :methodConfiguration :methodRepoMethod)]
```

Comment:
why not `get-in`?

Reply:
This is probably my code. I kinda like how it reads. But I'm starting to change my mind.